File: Python/solution/solution2501_2600/solution2529.py
# Solution 2529

class Solution(object):
    def maximumCount(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        neg = 0
        pos = 0
        for x in nums:
            if x < 0:
                neg += 1
            elif x > 0:
              pos += 1
        return max(neg, pos)

File: Rust/src/solution/solution2501_2600/solution2529.rs
use std::cmp::max;

// This is solution 2529 code
impl Solution {
    pub fn maximum_count(nums: Vec<i32>) -> i32 {
        let mut neg = 0;
        let mut pos = 0;
        for &x in &nums {
            if x > 0 {
                pos += 1;
            }else if x < 0 {
                neg += 1;
            }
        }
        return neg.max(pos);
    }
}
